network-api/networkapi/wagtailpages/pagemodels/index.py

```diff
@@ -202,6 +203,9 @@ def generate_entries_set_html(self, request, *args, **kwargs):
                 pass
 
         start = page * page_size
+        if 'entries_offset_for_callout' in context:
+            start = start - 1
```

Comment:
If the BlogIndexPages initial entries were offset by the callout box (for example setting initial entry list size from 12 -> 11), we then need to update the starting point of the next page.

If we dont, the next page is going to start at 13, leaving entry #12 out of the rendered list.

Reply:
**Question:** Do we use callout boxes in other index pages than the blog index? If not, we should not make this change on the index page, but rather keep it to the blog index.

Reply:
Moving this to the blog index would also mean you don't need to pull in the context that you don't really need here.

Reply:
I think it would be good to add some tests around this to cover all the edge cases, but I think in general this approach (of adjusting the start point) is fine.